.travis.yml

```diff
@@ -1,16 +1,17 @@
 sudo: false
 language: node_js
 node_js:
-  - '0.10'
+  - '5.10.1'
```

Comment:
That's good point thou. I did not knew that Node has LTS as well. We should try lowering Node version to 4.x and see if build still passes